segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/AttributeUpdateType.java

```diff
@@ -47,8 +47,8 @@
     /**
      * Any updates will replace the current attribute value, but only if the existing value matches
      * an expected value. If the value is different the update will fail. This can be used to
-     * perform compare and set operations. The value {@link SegmentMetadata#NULL_ATTRIBUTE_VALUE} is
-     * used to indicate a non value. IE: to remove the attribute or if the value is expected not to
+     * perform compare and set operations. The value {io.pravega.segmentstore.server.SegmentMetadata.NULL_ATTRIBUTE_VALUE}
```

Comment:
Why are you removing the @link tag? I think we need to add a # in front of the constant.

http://www.oracle.com/technetwork/articles/java/index-137868.html

Reply:
It complains that it doesn't find it. That's because the "segmentstore.server" package is not accessible from this package.

Reply:
I think that warning is incorrect. It is fine for this to refer to such a class.